app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class LikesController < ApplicationController
+  def creates
+    @review = Review.find(params[:review_id])
+    @like = Like.create(user_id: current_user.id, review: @review)
+    @review.like.to_i =  1
+    @trail = Trail.find(params[:trail_id])
+    if @like.save
+      flash[:notice] = "You liked this review"
+      redirect_to @trail
+    else
+      flash.now[:warning] = "Like wasn't added"
+      redirect_to @trail
+    end
+  end
+end
```

Comment:
Hmmm I dont think this is being used at all. You already perform this action in your API::LikesController, so it shouldnt be needed here.

Additionally, `creates` should be `create`. I imagine this is either not being hit or that it is buggy ಥ_ಥ

Reply:
Was not being used. File deleted.